README.md

```diff
@@ -47,27 +38,29 @@ Add the SDK package to your `package.json` file.
 
 `npm install @pngme/react-native-sms-pngme-android@2.0.4 --save`
 
-### _Step 2_
-Add your **SDK Token** to the project. In the sample app, the **SDK Token** is injected via the `.env` file:
+### Step 2
+
+Add your **SDK Token** to `.env`.
 
 ```text
 PNGME_SDK_TOKEN=XXXXXXXXXX
 ```
 
-> ⚠️ We recommend that additional measures be taken to protect the **SDK Token** when implementing in a production app.
+> ⚠️ We recommend that additional measures be taken to protect the **SDK Token** when implementing in a production app. Consider using an encrypted secrets manager to store the SDK token.
```

Comment:
Can we recommend some readings or a way for an individual to search for these details?

Reply:
Like a link to AWS documentation on Secrets manager or something

Reply:
Added 👍